docs/source/api.rst

```diff
@@ -125,3 +125,67 @@ For example::
 
 
 
+Detailed API reference
+------------------------
+Module: :mod:`traitlets.traitlets`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. automodule:: traitlets.traitlets
+    :members:
+    :private-members:
+
+Module: :mod:`traitlets.config.configurable`
```

Comment:
individual traitlets.config.x submodules should not be included, only the top-level `traitlets.config` should be considered public API